CHANGELOG

```diff
@@ -173,7 +173,10 @@ Version 4.2.1 (development)
   mixed meshes. The LOR Transfer miniapp (miniapps/tools/lor-transfer.cpp) now
   supports meshes with any element geometry.
 
-- Gitlab CI: use Spack (and Uberenv) to automate the build of TPLs.
+- Testing improvements:
+  * Transitioned from Travis to GitHub Action for testing/CI on GitHub.
+  * Use Spack (and Uberenv) to automate building of TPLs in LLNL GitLab tests.
```

Comment:
I think: "automate the build of TPLs" or "automate building TPLs". Am I wrong?

Reply:
How about this?

```suggestion
  * Use Spack (and Uberenv) to automate TPL building in LLNL GitLab tests.
```